.github/workflows/cla.yml

```diff
@@ -0,0 +1,26 @@
+name: CLA Checking
+on:
+  pull_request:
```

Comment:
We just want the `open` action to trigger this, right? [By default, `pull_request` includes other triggers as well](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request).

Reply:
Ah, my bad! I think I didn't catch this because it works similar to how we want by default:

> For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated.

But good to have this only when a new PR created. Fixing in next commit.